ebu_tt_live/bindings/validation/timing.py

```diff
@@ -196,6 +196,12 @@ def _semantic_postprocess_timing(self, dataset, element_content):
                     # Propagate the longest end time among the children
                     self._computed_end_time = max(children_computed_end_times)
 
+        if begin_timedelta is not None \
+        and end_timedelta is not None \
+        and begin_timedelta > end_timedelta:
+            self._computed_end_time = None
+            self._computed_begin_time = timedelta(0)
```

Comment:
I was expecting to ignore the element altogether if that is possible, so completely not sure if this is the right approach. I can see that it would work though, so let's leave like this for now...